chroma-agent/chroma_agent/agent_client.py

```diff
@@ -279,7 +279,10 @@ def terminate(self, plugin_name):
         else:
             daemon_log.info("SessionTable.terminate %s/%s" % (plugin_name, session.id))
             session.teardown()
-            del self._sessions[plugin_name]
+            try:
+                del self._sessions[plugin_name]
+            except KeyError:
+                daemon_log.warning("SessionTable.terminate session object already gone")
```

Comment:
It would be good to log the plugin_name here as well. I won't block for it though.